packages/multichain/src/caip25Permission.test.ts

```diff
@@ -0,0 +1,754 @@
+import type { NonEmptyArray } from '@metamask/controller-utils';
+import type { CaveatConstraint } from '@metamask/permission-controller';
+import {
+  CaveatMutatorOperation,
+  PermissionType,
+} from '@metamask/permission-controller';
+
+import type { Caip25CaveatValue } from './caip25Permission';
+import {
+  Caip25CaveatType,
+  caip25EndowmentBuilder,
+  Caip25EndowmentPermissionName,
+  Caip25CaveatMutatorFactories,
+  removeScope,
+} from './caip25Permission';
+import * as ScopeAssert from './scope/assert';
+import * as ScopeAuthorization from './scope/authorization';
+
+jest.mock('./scope/authorization', () => ({
+  validateAndNormalizeScopes: jest.fn(),
+}));
+const MockScopeAuthorization = jest.mocked(ScopeAuthorization);
+
+jest.mock('./scope/assert', () => ({
+  assertScopesSupported: jest.fn(),
+}));
+const MockScopeAssert = jest.mocked(ScopeAssert);
+
+const { removeAccount } = Caip25CaveatMutatorFactories[Caip25CaveatType];
+
+describe('endowment:caip25', () => {
+  beforeEach(() => {
+    MockScopeAuthorization.validateAndNormalizeScopes.mockReturnValue({
+      normalizedRequiredScopes: {},
+      normalizedOptionalScopes: {},
+    });
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
```

Comment:
This has been happening automatically since https://github.com/MetaMask/core/pull/4417, so it is not needed:

``` suggestion
```

Reply:
https://github.com/MetaMask/core/pull/4784/commits/18d1112b95e756334fbf06d7dd5b9ec24a8783f7